server/src/main/java/org/apache/druid/server/ClientQuerySegmentWalker.java

```diff
@@ -163,12 +169,15 @@ public <T> QueryRunner<T> getQueryRunnerForIntervals(Query<T> query, Iterable<In
     final DataSource freeTradeDataSource = globalizeIfPossible(newQuery.getDataSource());
     // do an inlining dry run to see if any inlining is necessary, without actually running the queries.
     final int maxSubqueryRows = query.context().getMaxSubqueryRows(serverConfig.getMaxSubqueryRows());
+    final long maxSubqueryMemory = query.context().getMaxSubqueryMemoryBytes(serverConfig.getMaxSubqueryMemory());
```

Comment:
Lets undocument the server property since it will enable all q's running with the new query path. 
Also we should benchmark the new query path.